src/test_all.py

```diff
@@ -211,7 +211,16 @@ def natural_keys(text):
                     except KeyError:
                         raise Exception(f" * Looks like we have a repeated example here! "
                                         f"Merge outputs before removing duplicates. :-/ \n {instance}")
-
+                                        
+            # Make sure there are no link in instances
+            url_reg  = r'http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
+            instances = data['Instances']    
+            for instance in instances:
+                ck_url = re.findall(url_reg, instance['input'])
+                if ck_url:
+                    print(f'⚠️ WARNING: Looks like there is a link in the input: {ck_url}')
+                    break
```

Comment:
should this raise an exception (rather than `break`)?

Reply:
We have many tasks with links in the input and we need to change some of their definitions. After updating them, I'll change this scope to raise an exception.

Reply:
I see. Let's remember to address these definitely in near future.